explorer/scan/discovery_test.go

```diff
@@ -151,7 +151,7 @@ func TestDiscoverAssets(t *testing.T) {
 		inv := getInventory()
 		discoveredAssets, err := DiscoverAssets(context.Background(), inv, nil, recording.Null{})
 		require.NoError(t, err)
-		assert.Len(t, discoveredAssets.Assets, 3)
+		assert.Len(t, discoveredAssets.Assets, 4)
```

Comment:
I am not 100% clear why this changed.

Reply:
I think this happened because of my change that merged here https://github.com/mondoohq/cnquery/commit/15161230099466995a27fe0a9d0c434710ca07b9

We now discover k8s namespaces always. I guess it took effect yesterday because yesterday the providers go released. This test relies on the released providers and doesn't use the builtin.